Makefile

```diff
@@ -109,9 +109,8 @@ version:
 
 .PHONY: versioninfo
 versioninfo:
-	go install github.com/josephspurrier/goversioninfo/cmd/goversioninfo@v1.4.0; \
-	go run scripts/generate_versioninfo/main.go; \
-	go generate cmd/telegraf/telegraf_windows.go; \
+	go install github.com/josephspurrier/goversioninfo/cmd/goversioninfo@v1.4.0
+	go generate cmd/telegraf/telegraf_windows.go
```

Comment:
Should we maybe just get rid of the `versioninfo` target and remove it from the CI config seeing it is now being called when we package a zip?

Reply:
I thought about this, but wondered if someone wanted to build the binary locally with `make`, with running `make versioninfo` first would get the version+logo embedded, right? If we think the only target should be for the packages, zip in this case, I'm also cool with that, just wanted to not remove something without chatting first.

Reply:
sounds good to me!